README.adoc

```diff
@@ -4000,6 +4000,23 @@ When you've got keys that are not symbols stick to the hash rockets syntax.
 { :a => 1, 'b' => 2 }
 ----
 
+=== to_h over Hash[] [[to-h-over-hash]]
```

Comment:
Also not it :( Of two "bad→good" examples, one doesn't use (and doesn't need) `#to_h` at all.
I really believe it should say something along the lines

> **Avoid `Hash[]` constructor**
>
> `Hash::[]` was pre-Ruby 2.1 way of constructing hashes from arrays of key-value pairs, or from a flat list of keys and values. It has an obscure semantic and looks cryptic in code. Since Ruby 2.1, `Enumerable#to_h` can be used to construct a hash from a list of key-value pairs, and it should be preferred. Instead of `Hash[]` with a list of literal keys and values, just a hash literal should be preferred.

Or something like that.

Reply:
This is much better :+1: changed.